recipes/thrust/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+class ThrustConan(ConanFile):
+    name = "thrust"
+    license = "Apache License 2.0"
+    description = ("Thrust is a parallel algorithms library which resembles"
+                   "the C++ Standard Template Library (STL). Thrust's high-level "
+                   "interface greatly enhances programmer productivity while "
+                   "enabling performance portability between GPUs and multicore CPUs."
+                   "Interoperability with established technologies "
+                   "(such as CUDA, TBB, and OpenMP) facilitates "
+                   "integration with existing software.")
+    topics = ("conan", "thrust", "parallel", "header_only")
+    homepage = "https://thrust.github.io/"
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
```

Comment:
```suggestion
    no_copy_source = True
    options = {"thrust_device_system": ["cuda", "cpp", "omp", "tbb"]
    default_options = {"thrust_device_system": "cuda"}
```

Reply:
I'll go with `cpp` as default, as the other options may not be available. What do you think?

Reply:
I'd prefer following the upstream, CUDA is the default option there. Also, any other option will not result in a different package id.